media-gfx/graphviz/graphviz-2.40.1.recipe

```diff
@@ -0,0 +1,165 @@
+	cmd:dot_builtins$secondaryArchSuffix
+	cmd:dot$secondaryArchSuffix
+	cmd:dot2gxl$secondaryArchSuffix
+	#cmd:dotty$secondaryArchSuffix
+	cmd:edgepaint$secondaryArchSuffix
+	cmd:fdp$secondaryArchSuffix
+	cmd:gc$secondaryArchSuffix
+	cmd:gml2gv$secondaryArchSuffix
+	cmd:graphml2gv$secondaryArchSuffix
+	cmd:gv2gml$secondaryArchSuffix
+	cmd:gv2gxl$secondaryArchSuffix
+	cmd:gvcolor$secondaryArchSuffix
+	cmd:gvgen$secondaryArchSuffix
+	cmd:gvpack$secondaryArchSuffix
+	cmd:gvpr$secondaryArchSuffix
+	cmd:gvmap$secondaryArchSuffix
+	cmd:gvmap.sh$secondaryArchSuffix
+	cmd:gxl2dot$secondaryArchSuffix
+	cmd:gxl2gv$secondaryArchSuffix
+	#cmd:lneato$secondaryArchSuffix
+	cmd:mm2gv$secondaryArchSuffix
+	cmd:neato$secondaryArchSuffix
+	cmd:nop$secondaryArchSuffix
+	cmd:osage$secondaryArchSuffix
+	cmd:patchwork$secondaryArchSuffix
+	cmd:prune$secondaryArchSuffix
+	cmd:sccmap$secondaryArchSuffix
+	cmd:sfdp$secondaryArchSuffix
+	cmd:tred$secondaryArchSuffix
+	cmd:twopi$secondaryArchSuffix
+	cmd:unflatten$secondaryArchSuffix
+	lib:libcdt$secondaryArchSuffix = $libcdtVersionCompat
+	lib:libcgraph$secondaryArchSuffix = $libcgraphVersionCompat
+	lib:libgvc$secondaryArchSuffix = $libgvcVersionCompat
+	lib:libgvpr$secondaryArchSuffix = $libgvprVersionCompat
+	lib:liblab_gamut$secondaryArchSuffix = $liblab_gamutVersionCompat
+	lib:libpathplan$secondaryArchSuffix = $libpathplanVersionCompat
+	lib:libxdot$secondaryArchSuffix = $libxdotVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	graphviz${secondaryArchSuffix}_devel = $portVersion
+	devel:libcdt$secondaryArchSuffix = $libcdtVersionCompat
+	devel:libcgraph$secondaryArchSuffix = $libcgraphVersionCompat
+	devel:libgvc$secondaryArchSuffix = $libgvcVersionCompat
+	devel:libgvpr$secondaryArchSuffix = $libgvprVersionCompat
+	devel:liblab_gamut$secondaryArchSuffix = $liblab_gamutVersionCompat
+	devel:libpathplan$secondaryArchSuffix = $libpathplanVersionCompat
+	devel:libxdot$secondaryArchSuffix = $libxdotVersionCompat
+	"
+REQUIRES_devel="
+	graphviz$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libexpat$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libltdl$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:freetype_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	./autogen.sh
+	runConfigure ./configure CFLAGS=-D_BSD_SOURCE \
```

Comment:
`CPPFLAGS` should be used for definitions. They're used by both C and C++ compilers, and would avoid overriding vendor defaults `CFLAGS`

```diff
-	runConfigure ./configure CFLAGS=-D_BSD_SOURCE \
+	runConfigure ./configure CPPFLAGS=-D_BSD_SOURCE \

Reply:
Need to push that one :)